src/components/Todolist.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import heading from "./heading";
+class todolist extends Component {
+  state = {};
```

Comment:
you should initialize your state with a default value

It could be something like 

```javascript 
state = {
      tasks: [] // an array because I image that you will want to have multiple todo items
} 
```